modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -33,67 +38,25 @@ Some actions are not yet supported:
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
 
-[NOTE]
-====
-Canonical does not endorse or support {susemgr}.
-====
 
-== Prepare to Register {ubuntu} Clients
+== Prepare to Register
 
 Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
 
-// ifndef works for displaying content when building uyuni
-// ifdef does not show the content for uyuni
-// ifeval works for displaying content
 
-ifeval::[{uyuni-content} == true]
+//ifeval::[{suma-content} == true]
```

Comment:
Not sure, but this probably needs to be uncommented.

Reply:
You are right.